### ISSUE TITLE
Please verify ONNX v1.10.0 Release Candidate

### ISSUE THREAD

**Issue by rajeevsrao** [onnx/onnx-tensorrt#720]
Hello ONNX partner,
We have published (to TestPyPI) a Release Candidate package for the upcoming ONNX v1.10.0 release. Please help validate it and let us know if you encounter any issues. Note that the tentative release date for ONNX v1.10.0 is July 31, 2021. Thank you for your assistance!

```
pip install numpy protobuf typing-extensions>=3.6.2.1
pip install -i https://test.pypi.org/simple/ onnx==1.9.100
```

* 07/21/2021: RC1 to use for validation - [onnx 1.9.100](https://test.pypi.org/project/onnx/1.9.100/)

cc @kevinch-nv @rajeevsrao

**Comment by jiapei100** [comment]
Bugs:

```console
error: ‘onnx2trt_onnx::optimization’ has not been declared
```

It looks **onnx-tensorrt** is still based on **onnx** of 2 years ago ?

**Comment by kevinch-nv** [comment]
Yes, there are some current incompatibilities with the new ONNX versions in the `onnx2trt` binary we have in this repo. We are working on deprecating it, then we can move to a later ONNX version.

**Comment by rajeevsrao** [comment]
@jiapei100 in the meantime you can build and use onnx-tensorrt parser (library only, not the onnx2trt executable) for ONNX rel-1.10.0 by specifying `-DBUILD_LIBRARY_ONLY=1` during CMake.

**Comment by jiapei100** [comment]
It looks like an issue from [https://github.com/onnx/optimizer](https://github.com/onnx/optimizer). I should have installed both **onnx**, as well as **onnx_optimizer**.
